src/lib/utils/date.ts

```diff
@@ -1,2 +1,55 @@
 export const dateToString = (published: Date | string) =>
   new Date(published).toISOString().split("T")[0]
+
+// Define an array of month names
+export const months = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+]
```

Comment:
I appreciate the work put in here @saurabhburade... I'm cautious about taking this approach though because it does not translate. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

We should use the native `Intl.DateTimeFormat` object in JavaScript to generate the correct display names depending on the `locale`. Does that make sense, are you familiar with this object?